BUILDING.md

```diff
@@ -265,3 +266,8 @@ To exclude add the following in `settings.gradle` file ```startParameter.exclude
     [settings.gradle](https://github.com/opensearch-project/job-scheduler/blob/main/settings.gradle)
     [build.sh](https://github.com/opensearch-project/job-scheduler/blob/main/scripts/build.sh)
 
+## Updating documentation
+
+The available [OpenSearch plugins](https://opensearch.org/docs/latest/install-and-configure/install-opensearch/plugins/#available-plugins) and [Dashboards plugins](https://opensearch.org/docs/latest/install-and-configure/install-dashboards/plugins/#available-plugins) are listed on the documentation site. 
```

Comment:
Dashboards -> OpenSearch Dashboards

documentation site -> documentation website + link the page

Reply:
Done